src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -283,8 +283,8 @@ export class BaseEditCourseForm extends React.Component {
       runTypeModes,
     } = parsedTypeOptions;
     const disabled = courseInReview || !editable;
-    const showMarketingFields = !currentFormValues.type || !courseTypes[currentFormValues.type]
-      || courseTypes[currentFormValues.type].course_run_types.some((crt) => crt.is_marketable);
+    const showMarketingFields = (currentFormValues.type && courseTypes[currentFormValues.type]) === undefined ? false
+      : courseTypes[currentFormValues.type].course_run_types.some((crt) => crt.is_marketable);
```

Comment:
nit: convert this to a function and add comment/jsdoc to explain the criteria for displaying marketing fields.